src/Sylius/Bundle/AdminBundle/Controller/ImpersonateUserController.php

```diff
@@ -34,23 +33,18 @@ final class ImpersonateUserController
     /** @var UserProviderInterface */
     private $userProvider;
 
-    /** @var RouterInterface */
-    private $router;
-
     /** @var string */
     private $authorizationRole;
 
     public function __construct(
         UserImpersonatorInterface $impersonator,
         AuthorizationCheckerInterface $authorizationChecker,
         UserProviderInterface $userProvider,
-        RouterInterface $router,
```

Comment:
BC break

Reply:
I removed this because, this is never used in this Controller.
I can revert this change ;)

Reply:
I totally get it, but we can't remove it :(

Reply:
Ok, i rollback this change :)